tests/foreman/ui/test_host.py

```diff
@@ -2451,3 +2465,46 @@ def test_positive_host_registration_with_non_admin_user(
 
         result = rhel8_contenthost.execute(cmd)
         assert result.status == 0, f'Failed to register host: {result.stderr}'
+
+
+@pytest.mark.tier2
+def test_all_hosts_delete(session, target_sat, function_org, function_location, new_host_ui):
+    """Create a host and delete it through All Hosts UI
+
+    :id: 42b4560c-bb57-4c58-928e-e5fd5046b93f
+
+    :expectedresults: Successful deletion of a host through the table dropdown
+
+    :CaseComponent:Hosts-Content
+
+    :Team: Phoenix-content
+
+    :CaseLevel: System
+    """
+    host = target_sat.api.Host(organization=function_org, location=function_location).create()
```

Comment:
I would suggest to use host name during create call, so that later we can use same name to verify host is being delete or not.
```suggestion
    host = target_sat.api.Host(organization=function_org, location=function_location, name="<any host name>").create()
```

Reply:
same as above, it's being checked in the entity that the delete or bulk delete was successsful.